src/AccessibilityInsights.SetupLibrary/FixedConfigSettingsProvider.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+
+namespace AccessibilityInsights.SetupLibrary
+{
+    public class FixedConfigSettingsProvider
+    {
+        public string ConfigurationFolderPath { get; }
+        public string UserDataFolderPath { get; }
+
+        public FixedConfigSettingsProvider(string configurationFolderPath, string userDataFolderPath)
+        {
+            this.ConfigurationFolderPath = configurationFolderPath;
+            this.UserDataFolderPath = userDataFolderPath;
+        }
+
+        public static FixedConfigSettingsProvider CreateDefaultSettingsProvider()
+        {
+            var sUserDataFolderPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.MyDoc‌​uments), "AccessibilityInsights");
```

Comment:
>sUserDataFolderPath [](start = 16, length = 19)

Not a fan of the "s" prefix here. Feels too much like we're trying to slip back to Reverse Hungarian Notation